packages/metrics-server/package.yaml

```diff
@@ -0,0 +1,5 @@
+url: https://github.com/helm/charts.git
+subdirectory: stable/metrics-server
+type: git
+commit: 61aca68bd482406a3c34e89732a86f24794d67b1
+packageVersion: 2.11.1 
```

Comment:
Also packageVersion stands for rancher patches, so in here it should be some numbers like `00` or `-rancher1`. So that the chart version will be `v1.2.300` or `v1.2.3-rancher1`. I would suggest adding 00 first.